extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -70,6 +70,35 @@ export async function launchDeployAppIntegrationQuickpick(project: Project): Pro
 	};
 }
 
+async function launchEulaQuickPick(baseImage: string): Promise<boolean> {
+	const baseImages = uiUtils.getDockerBaseImages();
+	const imageInfo = baseImages.find(x => x.name === baseImage);
+	let result: string | undefined = constants.openEulaString;
+	if (imageInfo?.agreementInfo?.link) {
+		const link = imageInfo?.agreementInfo?.link;
+
+		// Ask user to update app settings or not
+		//
+		let choices = [
+			constants.yesString,
+			constants.noString,
+			constants.openEulaString
```

Comment:
It would be cleaner to create a quick pick and then set a button to do this - https://github.com/microsoft/vscode-extension-samples/blob/136822998740e9f019d06f912b3b6f06fd49b484/quickinput-sample/src/multiStepInput.ts#L208